includes/class-comment.php

```diff
@@ -56,6 +59,7 @@ public static function register_comment_types() {
 			array(
 				'label'       => __( 'Tag', 'webmention' ),
 				'description' => __( 'Tags or tagging refers to categorizing or labeling content, your own or others (tag-reply), with words, phrases, names, or other information, optionally linked to specific people, events, locations, such as the practice of tagging posts being about certain people (person-tag), like tagging people or other items where (area-tag) they\'re depicted in a photo.', 'webmention' ),
+				'icon' => '📌',
```

Comment:
https://developer.wordpress.org/resource/dashicons/#tag

Reply:
maybe use https://developer.wordpress.org/resource/dashicons/#admin-post instead because of "bookmark" activity